src/pages/EmergencyNumbers.jsx

```diff
@@ -0,0 +1,114 @@
+import { PhoneIcon } from '@chakra-ui/icons';
+import React, { useEffect, useState } from 'react';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'useState' is defined but never used.